playground/TestPlatform.Playground/TestPlatform.Playground.csproj

```diff
@@ -62,7 +62,7 @@
       <FileToCopy Include="$(SourcePath)testhost\bin\$(Configuration)\net48\win7-x64\**\*.*" SubFolder="TestHostNetFramework\" />
 
       <!-- copy datacollectors" -->
-      <FileToCopy Include="$(SourcePath)datacollector\bin\$(Configuration)\net472\**\*.*" SubFolder="vstest.console\" />
+      <FileToCopy Include="$(SourcePath)datacollector\bin\$(Configuration)\net472\**\*.*" SubFolder="" />
```

Comment:
Could you explain this change?

Reply:
Fix the debuggability of the playground, that folder is wrong.